pysal/contrib/README.rst

```diff
@@ -52,7 +52,10 @@ Currently the following contribs are available:
 
     - .. versionadded:: 1.5
     - Path: pysal.contrib.viz
-    - Requires: `matplotlib`_
+    - Requires: `matplotlib`_, `pandas`_, `bokeh`_
+    - Optional:
```

Comment:
Not sure if this is the best way to specify it but I wanted a way to show you don't need `folium` or `geojson` if all you want to do is plot on  the `matplotlib` backend, for instance.